src/api/store/page_settings__aboutus.py

```diff
@@ -38,13 +38,23 @@ def create_about_us_page_setting(self, args) -> (dict, MassEnergizeAPIError):
   def update_about_us_page_setting(self, args) -> (dict, MassEnergizeAPIError):
     try:
       about_us_page_id= args.get('id', None)
+      page_image = args.pop('image', None)
+
       if about_us_page_id:
-        actions_page_setting = AboutUsPageSettings.objects.filter(id=about_us_page_id)
-        actions_page_setting.update(**args)
-        if not actions_page_setting:
+        page_setting = AboutUsPageSettings.objects.filter(id=about_us_page_id)
+        if not page_setting:
           return None, InvalidResourceError()
+        page_setting.update(**args)
```

Comment:
this will error out if there is anything in the args that is not consistent with a column in the database etc.  so to be sure its good to use, let's make sure the handler method cleans up the args real good.

Reply:
I agree with you, but here is a case of a lot of duplicated code.  The very same code is in page_settings__actions, page_settings__contactus, page_settings__donate, page_settings__teams, not of which do any cleaning up of args in the handler method.  
I was thinking of refactoring a bit since these models are basically the same, so the same code should check the args etc.